modules/ocf_irc/manifests/biboumi.pp

```diff
@@ -1,7 +1,13 @@
 class ocf_irc::biboumi {
-  ocf::repackage { 'biboumi':
-    backport_on => ['stretch'],
-    require     => Package['prosody'],
+  if $::lsbdistcodename != 'buster' {
```

Comment:
I don't think you should need this conditional at all, [`ocf::repackage` should do the right thing here](https://github.com/ocf/puppet/blob/2c7e5f4b7236617717b5f974d18544e518573fb6/modules/ocf/manifests/repackage.pp#L31-L33) and just not install the backported one on buster but still install the backport on stretch. Did it not work?